Master RMarkdown Document & Render Code/SDC Flag.R

```diff
@@ -0,0 +1,293 @@
+##### LOCALITY PROFILES MASTER DOC RENDER CODE #####
+
+library(readxl)
+library(janitor)
+library(tidyverse)
+library(knitr)
+library(markdown)
+library(rmarkdown)
+library(writexl)
+
+rm(list = ls())
+
+# system unmask function so files have read-write permissions
+Sys.umask("006")
+
+# Set file path
+lp_path <- "/conf/LIST_analytics/West Hub/02 - Scaled Up Work/RMarkdown/Locality Profiles/"
+
+# Source in functions code
+source("Master RMarkdown Document & Render Code/Global Script.R")
+
+# Below creates locality list of all the localities in a chosen HSCP
+lookup <- read_in_localities()
+
+#Add HSCP name
+
+HSCP <- "Orkney Islands"
+
+# Create list of localities in chosen HSCP
+locality_list <- lookup %>%
+  filter(hscp2019name == HSCP) %>%
+  pull(hscp_locality)
+
+
+## Loop to create the relevant dataframes for all the localities in the list
+
+
+# Initialize excel_output as a named list to store output data frames
+df_names <- c("Preventable_admission_PPA", "Emergency_Admissions",
+              "Unscheduled_Bed_Days", "Readmissions", 
+              "AE_attendances", "Delayed_Discharge", "Fall_Admissions", "MH_bed_days")
+
+excel_output <- setNames(vector("list", length(df_names)), df_names)
+
+# 2. Loop through each locality to create the appended excel output
+for (LOCALITY in locality_list) {
+  ## 2a) Source in all the scripts for a given LOCALITY
+  
+  # unscheduled care
+  source("./Unscheduled Care/2. Unscheduled Care outputs.R")  
+    
+    ppa_areas <- ppa_areas |> 
+      mutate(name = 'PPA')
+    emergency_adm_areas <- emergency_adm_areas |> 
+      mutate(name = 'emergency admissions')
+    bed_days_areas <- bed_days_areas |> 
+      mutate(name = 'Bed days')
+    readmissions_areas <- readmissions_areas |> 
+      rename('n' = 'read_28') |> 
+      mutate(name = 'readmissions')
```

Comment:
I think doing this here, would make the code easier to read as then in the list you could just do:

```r 
df <- list(...,
           "Readmissions" = readmissions_areas,
           ...
           )
```

```suggestion
    readmissions_areas <- readmissions_areas |> 
      filter(location == LOCALITY) |>
      select("financial_year", "n" = "read_28") |> 
      mutate(name = "readmissions")
```

Doing the filter and select earlier also means smaller objects in the environment.